tests/Mock/FetchTransactionPayPalFailure.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this is the same as FetchTransactionFailure.xml, so we don't need to make this a separate test